### PR TITLE
Support nested options

### PR DESCRIPTION
**Pull request by jonathantneal**
Does the current syntax make sense? #2

### CODE REVIEW

**Comment by ai** [comment]
We can split nested props by dot: `one.two: 1`

**Comment by TrySound** [comment]
@ai +1

**Comment by jonathantneal** [comment]
@ai, that looks really clever, but splitting properties with a dot `.` is not very CSS, IMHO.

**Comment by ai** [comment]
@jonathantneal also siggested

```css
one: {
  two: 2
}
```

File: index.js [hunk 1]
```diff
@@ -8,6 +8,24 @@ function trim (value) {
     return value.trim();
 }
 
+function pluginOptsFromRule (rule) {
+    var object = {};
```
Comment (ben-eb) [inline index.js]: Would prefer `options` here, as its more descriptive.

Comment (jonathantneal) [inline index.js]: Sure thing. It was named this to honor the existing `pluginOpts` variable name.

**Pull request merged**